app/controllers/at_who_controller.rb

```diff
@@ -32,7 +32,7 @@ def menu_items
   end
 
   def rep_items
-    repository = Repository.find_by_id(params[:repository_id])
+    repository = Repository.find_by_id(params[:repository_id]) || Repository.active.accessible_by_teams(@team).first
```

Comment:
Maybe we can call here `Repository.active.accessible_by_teams.find_by(id: params[:repository_id]) || Repository.active.accessible_by_teams(@team).first`
And then we do not need `can_read_repository?` permission check